frontend/src/uxbox/main/ui/settings/profile.cljs

```diff
@@ -90,14 +91,23 @@
                          :field :email}]
 
      [:span.settings-label (t locale "settings.profile.lang")]
-     [:select.input-select {:value (or (:lang data) "en")
+     [:select.input-select {:value (:lang data)
```

Comment:
@wizzytod A bit surprise by this modification... I'm guessing you had a merge / rebase issue there ?

Reply:
no issues - lang gets its default value on profile fetched